src/app/static/src/js/components/runReportTabs/runReportTabs.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+    <div id="report-log-container" class="mt-3">
+        <div id="report-key">
+            <div class="col-auto">
+            <running-reports-details id="report-details" :reportKey="reportKey"></running-reports-details>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script lang="ts">
+    import Vue from "vue"
+    import runningReportsDetails from "./runningReportsDetails.vue";
+
+    export default Vue.extend({
+        name: "runReportTabs",
```

Comment:
..I think this may have come about through some miscommunication over the [design doc](https://docs.google.com/presentation/d/1-n6wvgEEg8EWWzj6-uMG_5lHtATFv3vz2lm1gkdwObA/edit#slide=id.p) where the top level component there actually does contain both tabs (these to be moved from the ftl template), so that all Vue components can communicate via that top level comp. This is what Nick is working on just now, so this branch, or its code, will need to be modified in response to that.